internal/util/windowspriority/set_priority_windows.go

```diff
@@ -0,0 +1,55 @@
+//go:build windows
+// +build windows
+
+package windowspriority
+
+import (
+	"fmt"
+	"iter"
+	"maps"
+	"strings"
+
+	"golang.org/x/sys/windows"
+)
+
+// This is the default value in the alloycli package. We don't need constants
+// for the other values because they are not used in code.
+const PriorityNormal = "normal"
+
+var priorityMap = map[string]uint32{
+	"above_normal": windows.ABOVE_NORMAL_PRIORITY_CLASS,
+	"below_normal": windows.BELOW_NORMAL_PRIORITY_CLASS,
+	"high":         windows.HIGH_PRIORITY_CLASS,
+	"idle":         windows.IDLE_PRIORITY_CLASS,
+	PriorityNormal: windows.NORMAL_PRIORITY_CLASS,
+	"realtime":     windows.REALTIME_PRIORITY_CLASS,
+}
+
+func PriorityValues() iter.Seq[string] {
+	return maps.Keys(priorityMap)
+}
+
+func TranslatePriority(priorityString string) (uint32, error) {
+	priority, ok := priorityMap[strings.ToLower(priorityString)]
+	if !ok {
+		return 0, fmt.Errorf("invalid priority: %s", priorityString)
+	}
+	return priority, nil
+}
+
+func SetPriority(priorityString string) error {
+	if priorityString == "" {
+		return nil
+	}
```

Comment:
Why not omit this and let `TranslatePriority` return an error?

Reply:
I think it's a leftover artifact of when I was attempting to have the main code be cross-platform and the os-specific implementation be smaller. Will clean up.